scripts/xtensa-build-zephyr.sh

```diff
@@ -189,8 +189,11 @@ build_platforms()
 				XTENSA_CORE="X6H3CNL_2017_8"
 				XTENSA_TOOLS_VERSION="RG-2017.8-linux"
 				;;
-			icl)
+			icl|jsl)
 				PLAT_CONFIG='intel_adsp_cavs20'
+				if test $platform = jsl ; then
```

Comment:
Does this one pass shellcheck? Sometimes it's smart enough to know quotes are not needed, sometimes it's not.

Reply:
What's shellcheck? :) I copied the tgl-h check and that fails as well. I'll update.

Reply:
> What's shellcheck? :)

:-)

The best thing since sliced bread.

The only way to keep shell scripts relatively maintainable and trouble-free.